bin/pkg/validate_lock_dir.ml

```diff
@@ -70,17 +68,11 @@ let validate_lock_dirs ~context_name_arg ~all_contexts_arg =
 ;;
 
 let term =
-  let+ builder = Common.Builder.term
-  and+ context_name =
-    Pkg_common.context_term ~doc:"Validate the lockdir associated with this context"
-  and+ all_contexts =
-    Arg.(value & flag & info [ "all-contexts" ] ~doc:"Validate all lockdirs")
-  in
+  let+ builder = Common.Builder.term in
   let builder = Common.Builder.forbid_builds builder in
+  (* CR-someday alizter: This should accept the Pkg_common.Lock_dirs.term *)
```

Comment:
This command can also be improved to accept lock directories, but it will need some more refactoring and I would rather do that in another PR.